src/Environment.cpp

```diff
@@ -317,3 +335,18 @@ Environment::~Environment()
 		delete[] envp[i];
 	}
 }
+
```

Comment:
Is this overloading doing the same like the printVars method in this object? Do I read it correctly?

Reply:
It's correct, I had the print before Daniil started using the overload operator which is more elegant, I erased the print method